help/en/contributing_translating.html

```diff
@@ -74,7 +74,7 @@ <h3>Translating documentation</h3>
 
 <p>Start translating. It's best to translate table of contents first and proceed with actual content later. To translate table of contents open menu.xml file in your editor of choice and translate values of every <em>text</em> attribute. E.g. for &lt;area text="Setting up Luminance HDR" file="prefs.html"&gt; translatable text will be "Setting up Luminance HDR".</p>
 
-<p>To test your translation open a terminal window, go to the top level directory with source code, and run <tt>sudo make install</tt> to reinstall Luminance HDR. All available translations will be automaticaly copied to the right place, and you will have to restart Luminance HDR to let it pick the added translation. However, as you progress with your translation, you only need to restart help browser to see changes in table of contents and you don't even need to restart the help browser to see changes in separate chapters&nbsp;&mdash; just click on some other chapter and go back again.</p>
+<p>To test your translation open a terminal window, go to the top level directory with source code, and run <tt>sudo make install</tt> to reinstall Luminance HDR. All available translations will be automatically copied to the right place, and you will have to restart Luminance HDR to let it pick the added translation. However, as you progress with your translation, you only need to restart help browser to see changes in table of contents and you don't even need to restart the help browser to see changes in separate chapters&nbsp;&mdash; just click on some other chapter and go back again.</p>
```

Comment:
github ui is not very clear here but the specific modification is `s/automaticaly/automatically/`